### PR TITLE
Maps in subuniverses

### PR DESCRIPTION
**Pull request by fredrik-bakke**
A simple PR that moves "maps in subuniverses" to their own file and formalizes that they are closed under base change.

### CODE REVIEW

File: src/foundation/global-subuniverses.lagda.md [hunk 1]
```diff
@@ -67,41 +67,29 @@ record global-subuniverse (α : Level → Level) : UUω where
...
-We say a map is _in_ a global subuniverse if each of its
-[fibers](foundation-core.fibers-of-maps.md) is.
+open global-subuniverse public
 
-```agda
 module _
   {α : Level → Level} (P : global-subuniverse α)
-  {l1 l2 : Level} {A : UU l1} {B : UU l2}
   where
-
```
Comment (VojtechStep) [inline src/foundation/global-subuniverses.lagda.md:100]: This `module` declaration is now empty

File: src/foundation/maps-in-global-subuniverses.lagda.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  is-in-global-subuniverse-map-base-change :
+    is-in-global-subuniverse-map 𝒫 f →
+    cartesian-hom-arrow g f →
+    is-in-global-subuniverse-map 𝒫 g
+  is-in-global-subuniverse-map-base-change F α d =
+    is-closed-under-equiv-global-subuniverse 𝒫 (l1 ⊔ l2) (l3 ⊔ l4)
+    ( fiber f (map-codomain-cartesian-hom-arrow g f α d))
+    ( fiber g d)
+    ( inv-equiv (equiv-fibers-cartesian-hom-arrow g f α d))
+    ( F (map-codomain-cartesian-hom-arrow g f α d))
```
Comment (VojtechStep) [inline src/foundation/maps-in-global-subuniverses.lagda.md:105]:
```suggestion
    is-closed-under-equiv-global-subuniverse 𝒫 (l1 ⊔ l2) (l3 ⊔ l4)
      ( fiber f (map-codomain-cartesian-hom-arrow g f α d))
      ( fiber g d)
      ( inv-equiv (equiv-fibers-cartesian-hom-arrow g f α d))
      ( F (map-codomain-cartesian-hom-arrow g f α d))
```

**Review by VojtechStep — CHANGES_REQUESTED** [review]
I just have two minor nits, after which we can merge this PR 👍

**Comment by fredrik-bakke** [comment]
Sorry, I had an unpushed minor fix that fits with one of the other changes I made in this PR.

**Review by VojtechStep — APPROVED** [review]
No problem, looks good 👍

**Pull request merged**